ipaserver/install/ipactl.py

```diff
@@ -604,5 +715,5 @@ def main():
         ipa_status(options)
 
 
-if __name__ == '__main__':
-    installutils.run_script(main, operation_name='ipactl')
+if __name__ == "__main__":
+    installutils.run_script(main, operation_name="ipactl")
```

Comment:
Is there a reason for switching from single to double quotes?

Reply:
Yes, codestyle changes are mandatory otherwise pylint will complain.
I used python3-black for most of the code changes and it seems to prefer double quotes.